concordium-sdk/src/test/testresources/smartcontractschema/lib.rs

```diff
@@ -0,0 +1,142 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+//! # A Concordium V1 smart contract. This contract is only for genereating a schema to test serialization of `SchemaParameter` and is never meant to be excecuted, hence the dummy state/error/init.
+use concordium_std::*;
+
+/// Your smart contract state.
+#[derive(Serialize, SchemaType)]
+pub struct State {
+    // Dummy state.
+}
+
+/// Dummy init functions
+#[init(contract = "java_sdk_schema_unit_test")]
+fn init<S: HasStateApi>(
+    _ctx: &impl HasInitContext,
+    _state_builder: &mut StateBuilder<S>,
+) -> InitResult<State> {
+    // Your code
+
+    Ok(State {})
+}
+
+#[derive(Serialize, SchemaType)]
+#[concordium(transparent)]
+struct ListParam (
+    Vec<ContractAddress>
+);
+
+#[derive(Serialize, SchemaType)]
+struct AbstractAddressContainer {
+    address:   Address,
+}
+
+#[derive(Serialize, SchemaType)]
+struct AccountAddressContainer {
+    address:   AccountAddress,
+}
+
+#[derive(Serialize, SchemaType)]
+struct ContractAddressContainer {
+    address:   ContractAddress,
+}
+
+// The `ParseError` in the following result types is never thrown, as the functions only exist to generate a schema for serialization of SC parameters.
+
+/// Takes ListParam as parameter for ensuring correct serialization of ListParam
```

Comment:
```suggestion
/// Takes [`ListParam`] as parameter for ensuring correct serialization of [`ListParam`]
```
Please use proper rust doc: https://doc.rust-lang.org/rustdoc/write-documentation/linking-to-items-by-name.html

Reply:
As far as i read this is for linking to other rustdcoc? ListParam is refering to the Java class

Reply:
Ahh. maybe clarify that.